src/main/java/io/nats/client/api/ConsumerConfiguration.java

```diff
@@ -33,15 +34,16 @@
  */
 public class ConsumerConfiguration implements JsonSerializable {
 
-    private static final Duration MIN_ACK_WAIT = Duration.ofNanos(1);
-    private static final Duration MIN_DEFAULT_IDLE_HEARTBEAT = Duration.ZERO;
-    private static final Duration DEFAULT_ACK_WAIT = Duration.ofSeconds(30);
-    private static final long DEFAULT_MAX_PULL_WAITING = 512;
+    public static final Duration MIN_ACK_WAIT = Duration.ofNanos(1);
+    public static final Duration MIN_DEFAULT_IDLE_HEARTBEAT = Duration.ZERO;
+    public static final Duration DEFAULT_ACK_WAIT = Duration.ofSeconds(30);
+    public static final long DEFAULT_MAX_PULL_WAITING = 512;
```

Comment:
No reason for these values to be private. I realized this when I was building unit tests.